mcodingbot/utils/peps.py

```diff
@@ -39,10 +41,19 @@ async def fetch_pep_info(self, bot: Bot) -> None:
                     )
                     for pep_id, pep in (await resp.json()).items()
                 }
+                self._pep_map = {
+                    k: f"{v.title} ({v.number})" for k, v in self._peps.items()
+                }
```

Comment:
Gonna need to think about this. I feel like this can be done without a second dict.

Reply:
The search function accepts a very specific format (either Collection[str] or Mapping[value, str]). Passing the original dict wouldn't work, and passing just the title would make it impossible to get the rest of the pep info from the response.

The dict either needs to be generated every time this function is called, or stored separately.

I could be missing something ofc.

Reply:
I spent a while on thinking of a better way to do this. Unfortunately this is the best way other than generating it every time it is called.